composer.json

```diff
@@ -9,7 +9,7 @@
         }
     ],
     "require-dev": {
-        "php": ">=7.0",
+        "php": ">=7.1",
```

Comment:
Just because variable unpacking is only available in 7.1 onwards, eg

https://github.com/mtrajano/laravel-swagger/blob/20823090a4b910fafe4f2431bbc1f0d4cfee1251/src/Generator.php#L134